src/ai-config.ts

```diff
@@ -0,0 +1,53 @@
+import { Message } from "whatsapp-web.js";
+import { aiConfigTarget, aiConfigTypes, aiConfigValues } from "./types/ai-config";
+
+const aiConfig = {
+    dalle: {
+        size: "512x512",
+    },
+    // chatgpt: {}
+}
+
+const handleMessageAICONFIG = async (message: any, prompt: any) => {
```

Comment:
handleMessageAIConfig

Reply:
This was due to the `handleMessageDALLE` and `handleMessageGPT`🤔